src/pages/studyView/StudyViewPage.tsx

```diff
@@ -172,6 +175,18 @@ export default class StudyViewPage extends React.Component<IStudyViewPageProps,
                              className="mainTabs">
 
                         <MSKTab key={0} id="summaryTab" linkText="Summary">
+                            {
+                                this.store.invalidSampleIds.length > 0 &&
+                                this.showErrorMessage &&
+                                <div>
+                                    <div className="alert alert-danger">
+                                        <button type="button" className="close" onClick={(event) => this.showErrorMessage = false}>&times;</button>
+                                        Following sample(s) might have been deleted/updated with the recent data updates
```

Comment:
I think the language can be easily updated base on the invalidSampleIds length.

Reply:
Also add "The ..."